src/components/progress-tracker/progress-tracker.pw.tsx

```diff
@@ -243,6 +246,23 @@ test.describe("Check props for ProgressTracker component", () => {
     );
   });
 
+  test(`should position current value label to the left of tracker line when labelsPosition prop is "left"`, async ({
```

Comment:
**nit-pick** (non-blockiing): as we're not using string interpolation here we can use speech marks for the test description instead of an apostrophe

Reply:
I know this is the same for the other tests around this one, happy to leave as is